fiftyone/core/stages.py

```diff
@@ -361,7 +373,12 @@ def _kwargs(self):
     def _params(cls):
         return [
             {"name": "field", "type": "str"},
```

Comment:
Should this have a placeholder? It's empty on my end. I think makePlaceholder might be intended to handle this, but it isn't:
![image](https://user-images.githubusercontent.com/3719547/93102994-119a7680-f67a-11ea-9152-8f576b07882c.png)